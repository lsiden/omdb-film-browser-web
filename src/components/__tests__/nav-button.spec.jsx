import React from 'react'
import TestRenderer from 'react-test-renderer';

import NavButton from 'components/nav-button'

const defaultProps = {
  onClick: jest.fn(),
}

const createWrapper = () => TestRenderer.create(
  <NavButton {...defaultProps}>Press Me</NavButton>,
)

test('click invokes onClick()', () => {
  expect(createWrapper()).toMatchSnapshot()
})
