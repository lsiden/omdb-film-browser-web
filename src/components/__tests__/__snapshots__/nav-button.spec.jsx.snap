// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`click invokes onClick() 1`] = `
<button
  className="btn btn-link"
  onClick={[MockFunction]}
  style={Object {}}
  type="button"
>
  Press Me
</button>
`;
